test/Microsoft.Azure.Devices.Edge.Test.Common/EdgeDevice.cs

```diff
@@ -14,20 +15,50 @@ public class EdgeDevice
         readonly IotHub iotHub;
         readonly bool owned;
 
-        EdgeDevice(Device device, bool owned, IotHub iotHub)
+        readonly NestedEdgeConfig nestedEdgeConfig;
+
+        public class NestedEdgeConfig
+        {
+            public readonly string DeviceHostname;
+            public readonly bool IsNestedEdge;
+            public readonly Option<string> ParentDeviceId;
+            public readonly string ParentHostname;
```

Comment:
these seem like they can just be readonly getters

Reply:
donezo